src/NUnitFramework/framework/Constraints/ConstraintExpression.cs

```diff
@@ -8,10 +8,10 @@
 // distribute, sublicense, and/or sell copies of the Software, and to
 // permit persons to whom the Software is furnished to do so, subject to
 // the following conditions:
-// 
+//
```

Comment:
@jnm2 mentioned that above, but too much noise building up on this PR 😸 

> I used the trailing whitespace remover extension recommended by @jnm2 so you may want to append `?w=1` to the file diff to reduce the noise.

Sorry, I probably wasn't very clear in my instructions. It is an old trick that @CharliePoole and I use, so thought it was obvious 😄